lte/gateway/configs/pipelined.yml_prod

```diff
@@ -152,4 +152,4 @@ uplink_gw_mac: 'ff:ff:ff:ff:ff:ff'
 
 he_proxy_eth_mac: 'e6:8f:a2:80:80:80'
 
-ovs_gtp_stats_polling_interval: 60
+ovs_gtp_stats_polling_interval: 180
```

Comment:
We don't seem to have an option to specify jitter in the base class. It will be nice to specify add that sometime. It is not a priority. It will help reducing the case of running all the timer callbacks synchronized at a time.

Reply:
Yeah this will be useful to add into the `Job` class, since we use it for many of the continuous operations on the magma services, we can add it as part of the BE work for future sprints.